src/xrpld/peerfinder/detail/Logic.h

```diff
@@ -360,7 +360,7 @@ class Logic
         beast::IP::Endpoint const& local_endpoint)
     {
         JLOG(m_journal.trace())
-            << beast::leftw(18) << "Logic connected" << slot->remote_endpoint()
+            << beast::leftw(18) << "Logic connected" << slot->fingerprint()
```

Comment:
```suggestion
            << beast::leftw(18) << "Logic connected " << slot->fingerprint()
```

I think an extra space is necessary after `Logic connected` to separate the fingerprint.